### PR TITLE
Apply global theme styles to default blog posts

### LINKED ISSUE #2688: Blog cards styles are not applying to placeholder images
**User Story**: As a merchant installing a new theme to use for my online store, I should be able to see an accurate representation of what my theme will look like with or without placeholder content. What I see by default, should reflect what I'll see when I upload a real image

**Ultimate Goal**: Achieve a 1:1 experience between placeholders images and real images

**More context**: With the introduction of new placeholder content, and fresh eyes on the topic, we noticed some inconsistencies between the empty and populated states with images. [Notes here](https://docs.google.com/document/d/1ZVYZsJmjRJAmKOTpWYbW9-dzrZ4nVmfSpFNSvj6Iakk/edit#)

## Identified Problem:
The styles of the blog cards are not applying to the placeholder images. 

## Expected behaviour
The default styles of blog cards placeholder images should appear with the same styles as a if they were populated with real blog images.

## Task
- [x]  Ensure styles applies to blog cards placeholder images

### PR DESCRIPTION
**Pull request by lougoncharenko**
### PR Summary: 

Default blog cards are responsive to presets and can be altered by number of blog posts to show, number of columns, color scheme, section paddings and show featured image.

Blog posts theme settings can be applied to default blog posts.

The placeholder image's height and width on default blog posts match other blog posts.

### Why are these changes introduced?

Fixes #2688 

### What approach did you take?
I used the article card and card class names to make the default blog cards responsive to the blog card theme settings and removed the blog placeholder specific classes that weren't responsive to the theme settings. 

### Visual impact on existing themes
This new change will not impact existing merchant stores but will improve the user experience when adding a featured blog. 

### Testing steps/scenarios

- [ ] Test default blog posts in mobile, tablet and desktop version
- [ ] Remove featured image in both standard and card blog posts
- [ ] Change number of blog posts to show in the presets
- [ ] Change number of columns on desktop in the presets

Blog card theme settings
- [ ] Test both styles: standard and card
- [ ] Test the text alignment: left, center and right
- [ ] Apply the color scheme
- [ ] Test border styles: thickness, opacity and corner radius.
- [ ] Test Shadow: opacity, horizontal and vertical offset and blur

### Demo links

- [Store](https://os2-demo.myshopify.com/?_ab=0&_fd=0&_sc=1)
- [Editor](https://os2-demo.myshopify.com/admin/themes/140152635414/editor?section=template--17290341023766__3a671bdc-15fc-4adc-a71d-87c7e3bc6731)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [x] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
   },
   "[liquid]": {
     "editor.formatOnSave": true
-  }
+  },
+  "prettier.useEditorConfig": true
```
Comment (ludoboludo) [inline .vscode/settings.json:12]: I don't think this should part of the PR 🤔

File: sections/featured-blog.liquid [hunk 2]
```diff
@@ -91,24 +91,47 @@
...
-                  <h2>
-                    {{ 'sections.featured_blog.onboarding_title' | t }}
-                  </h2>
-                  <p class="rte-width">
-                    {{ 'sections.featured_blog.onboarding_content' | t }}
-                  </p>
+              <div class="blog-container {% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
+                <div class="card article-card card--{{ settings.blog_card_style }}
+                    {% if settings.blog_card_style == 'card' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}
+                    {% if settings.blog_card_style == 'card' and section.settings.media_height == nil or section.settings.show_image == false %} ratio{% endif %}
```
[the remark this answers is not in the document]
Comment (lougoncharenko) [reply sections/featured-blog.liquid:97]: I removed this line and everything seems to be working just fine, so I think you're right that we don't need it.

File: sections/featured-blog.liquid [hunk 3]
```diff
@@ -91,24 +91,46 @@
                 style="--animation-order: {{ forloop.index }};"
```
Comment (ludoboludo) [inline sections/featured-blog.liquid:91]: Not related to this line or your PR. 
We're checking against the wrong setting on the wrapping `ul` element. We're using a conditional with `settings.card_style == 'standard'` which points to the product card settings. It should be `settings.blog_card_style == 'standard'`. 

Do you mind adding that to your PR ? It's a very small tweak 😅 Though usually we would want to do that in its own PR.

File: sections/featured-blog.liquid [hunk 4]
```diff
@@ -91,24 +91,46 @@
...
-                    {{ 'sections.featured_blog.onboarding_title' | t }}
-                  </h2>
-                  <p class="rte-width">
-                    {{ 'sections.featured_blog.onboarding_content' | t }}
-                  </p>
+              <div class="article-card-wrapper card-wrapper{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
+                <div class="card article-card card--{{ settings.blog_card_style }}
+                    {% if settings.blog_card_style == 'card' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}
+                    {% if section.settings.show_image %} card--media{% else %} card--text{% endif %}"
+                   style=" {% if settings.blog_card_style == 'standard' %} --ratio-percent: 100%;{% elsif settings.blog_card_style == 'card' %} --ratio-percent: 0%; flex-direction: column;{% endif %}"
```
Comment (ludoboludo) [inline sections/featured-blog.liquid:98]: Since we don't have the `ratio` classe anymore, you can remove the flex direction you have in here. It's already taken care of:


```suggestion
                   style=" {% if settings.blog_card_style == 'standard' %} --ratio-percent: 100%;{% elsif settings.blog_card_style == 'card' %} --ratio-percent: 0%; flex-direction: column;{% endif %}"
```

File: sections/featured-blog.liquid [hunk 5]
```diff
@@ -91,24 +91,46 @@
...
+                >
+                  <div class="card__inner{% if settings.blog_card_style == 'standard' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %} ratio"
+                    style="--ratio-percent: 80%;"
+                  >
+                    {%- if section.settings.show_image == true -%}
+                      <div class="article-card__image-wrapper card__media">
+                        <div class="article-card__image media media--hover-effect"
+                          {% if section.settings.media_height == 'adapt' %}
+                            style="padding-bottom: 100%;"
+                          {% endif %}
```
Comment (ludoboludo) [inline sections/featured-blog.liquid:108]: I don't think this is needed as there is a `section.settings.media_height` in the blog post section. 

So it never runs. Could be removed 👍

File: sections/featured-blog.liquid [hunk 6]
```diff
@@ -91,24 +91,46 @@
...
+                        >
+                          {{ placeholder_image | placeholder_svg_tag: 'blog-placeholder-svg' }} 300w
```
Comment (ludoboludo) [inline sections/featured-blog.liquid:110]: I think this `300w` isn't where it's supposed to be 
```suggestion
                          {{ placeholder_image | placeholder_svg_tag: 'blog-placeholder-svg' }}
```

File: sections/featured-blog.liquid [hunk 7]
```diff
@@ -91,24 +91,46 @@
...
+                    <div class="card__content">
+                      <div class="card__information">
+                        <h3 class="card__heading h2">
+                          {{ 'sections.featured_blog.onboarding_title' | t }}
+                        </h3>
+                        <p class="article-card__excerpt rte-width">
+                          {{ 'sections.featured_blog.onboarding_content' | t }}
+                        </p>
+                      </div> 
+                    </div>
```
Comment (ludoboludo) [inline sections/featured-blog.liquid:123]: We're missing a closing `</div>` I believe here 🤔

**Review by Oliviammarcello — APPROVED** [review]
I'm happy to review it again if there are any larger code changes that need to be done, otherwise it LGTM. 🎉

File: sections/featured-blog.liquid (hunk 3 above)
Comment (lougoncharenko) [reply sections/featured-blog.liquid:91]: I added this to my PR! It's such a small change, there's no reason to open a separate issue for it :D

**Review by kmeleta — APPROVED** [review]

File: sections/featured-blog.liquid [hunk 8]
```diff
@@ -91,25 +91,44 @@
...
-                  <div class="placeholder media media--hover-effect media--landscape">
-                    {{ placeholder_image | placeholder_svg_tag: 'placeholder-svg' }}
+              <div class="article-card-wrapper card-wrapper{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
+                <div class="card article-card card--{{ settings.blog_card_style }}
+                    {% if settings.blog_card_style == 'card' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}
+                    {% if section.settings.show_image %} card--media{% else %} card--text{% endif %}"
+                   style=" {% if settings.blog_card_style == 'standard' %} --ratio-percent: 100%;{% elsif settings.blog_card_style == 'card' %} --ratio-percent: 0%;{% endif %}"
+                >
+                  <div class="card__inner{% if settings.blog_card_style == 'standard' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %} ratio"
+                    style="--ratio-percent: 80%;"
```
Comment (ludoboludo) [inline sections/featured-blog.liquid:101]: Any specific reason why we use `80%` here instead of `100%` ? 

Here I set one image to use `100%` to compare what it looks like next to the `80%`:

**Review by ludoboludo — APPROVED** [review]
Looking good, just had one comment about the ratio value but otherwise looks good to go.

So feel free to adjust or ship as is if that was the intended height 👍

File: sections/featured-blog.liquid (hunk 8 above)
Comment (lougoncharenko) [reply sections/featured-blog.liquid:101]: when I spoke with @Oliviammarcello, she preferred the 80 :)
